iac-common/src/main/java/org/sonar/iac/common/api/tree/impl/TextRange.java

```diff
@@ -55,4 +55,9 @@ public boolean equals(Object other) {
   public int hashCode() {
     return Objects.hash(start, end);
   }
+
+  @Override
+  public String toString() {
+    return "[" + start.line() + ":" + start.lineOffset() + "/" + end.line() + ":" + end.lineOffset() + "]";
+  }
```

Comment:
Not used or?

Reply:
It is indirectly used in the `Verifier` class, so when one of our test fail, we get meaningful messages like this:
```
* [UNEXPECTED_ISSUE] at [43:8/43:41] with a message: "Restrict IP addresses authorized to access administration services."
* [NO_ISSUE] Expected but no issue on range [43:31/43:41].
```
Otherwise instead of coordinates we get `TextRange@{hash}` which is not really helpful :sweat_smile: